src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.ts

```diff
@@ -121,8 +121,10 @@ export class VehicleTechnicalRecordComponent implements OnInit, AfterViewInit {
     }
   }
 
-  createTest(isComplete?: string): void {
-    if (isComplete) {
+  createTest(techRecord?: TechRecordModel): void {
+    if (techRecord?.hiddenInVta) {
+      alert('Vehicle record is hidden in VTA.\n\nShow the vehicle record in VTA to start recording tests against it.');
```

Comment:
Would VSAs carrying out contingency test also see this message? Would we not want to check the user has permissions to update the `hiddenInVta` flag before showing that message?